README.md

```diff
@@ -23,83 +21,57 @@ Currently the operator installs the following products:
 - 3scale
 - Integreatly solution explorer
 
-## Prerequisites
+### managed-api
+
+- 3scale
+- RHSSO (both a cluster instance and a user instance)
+- Marin3r
 
-NOTE: Due to a change in how networking is configured for openshift in v4.4.6 (mentioned in the [cloud resource operator](https://github.com/integr8ly/cloud-resource-operator#supported-openshift-versions)) there is a limitation on the version of Openshift that RHMI can be installed on for BYOC clusters.
```

Comment:
This is being removed because we are past the point of the possibility of installations happening below certain versions. Upgrades don't matter because the vpc doesn't change on upgrades